src/scripts/install-chrome.sh

```diff
@@ -106,9 +106,9 @@ else
   # download chrome
   echo "Preparing Chrome installation for Debian-based systems"
   if [[ "$ORB_PARAM_CHROME_VERSION" == "latest" ]]; then
-    ENV_IS_ARM=$(! dpkg --print-architecture | grep -q arm; echo $?)
+    ENV_IS_ARM=$(dpkg --print-architecture | grep arm64)
```

Comment:
I just tried test running these changes and now most of the tests are failing, I think that this line should go back to how it was originally but replacing line 111 with `if [ "$ENV_IS_ARM" == "1" ]; then` would fix this issue